### DISCUSSION TITLE
Styling details

### DISCUSSION THREAD

**Discussion by mmpadilla15** [thegetty/quire#530] [Get Help] [answered]
Looking for some advice on some styling details that I can't figure out:

1. I want to make part of a title on a plate italicized, in this specific case, the "also called." I think I've added the proper markdown but it's not showing on the site. Here's the YML code:
![Screen Shot 2022-10-06 at 7 34 43 PM](https://user-images.githubusercontent.com/74433007/194442320-8a0480d5-c0f5-47b2-b82e-8d1cf60e18bf.png)
And here's the plate:
![Screen Shot 2022-10-06 at 7 33 14 PM](https://user-images.githubusercontent.com/74433007/194442388-31cb4457-d510-467a-bd8e-9a855eafec64.png)

2. In a figure caption, I need to italicize a closing bracket. In this case, it's the one after "Beach Log." I've tried several different things and none work - they usually break the site. This is my code in figures.yml:
![Screen Shot 2022-10-06 at 7 39 04 PM](https://user-images.githubusercontent.com/74433007/194442759-85f6447f-e060-4ee5-b009-549751001532.png)
And here's the caption on the site:
![Screen Shot 2022-10-06 at 7 39 54 PM](https://user-images.githubusercontent.com/74433007/194442793-bed9f401-7214-4efd-94e3-2556d1cec7ff.png)

TIA!

**Comment by geealbers** [comment, marked as answer]
For the first one, there's a confusing behavior in Quire with titles of objects that are displayed on entry pages, in that the title that's shown on the page is actually coming from the YAML at the top of the Markdown file for that page rather than from any title you might list in objects.yaml. My hunch is that you need to add the italics formatting (which otherwise looks correct) to the Markdown page.

For the second, this should work: `*[Charles T. Williams with his chainsaw sculpture* Beach Log *]*`. However, if not, you could also try using HTML tags instead of Markdown. Sometimes the markdown processor can get confused. The HTML version would be `<em>[Charles T. Williams with his chainsaw sculpture</em> Beach Log <em>]</em>`

**Reply by mmpadilla15** [reply]
@geealbers Thank you thank you!

1. That worked!
2. Your first suggestion was one I tried yesterday and it did not work. The closing bracket rendered as *]* instead of placing the bracket in italics. But the HTML solution worked.
